core/src/models/order.rs

```diff
@@ -40,6 +41,29 @@ impl Order {
             self.remaining_sell_amount,
         )
     }
+
+    pub fn to_element(&self, balance: primitive_types::U256) -> Element {
+        // Some conversions are needed because the primitive types crate is on different versions in
```

Comment:
I would say we should just pin the price graph version to be the same as core. Then we don’t need to do these conversions. It’s not really used anyway.

Reply:
This code is actually meant to be in the next PR. Cleaned up the mess and repushed the change.